mkosi/resources/man/mkosi.1.md

```diff
@@ -204,80 +204,67 @@ The following command line verbs are known:
+
+`--help`, `-h`
+:   Show brief usage information.
 
 `--force`, `-f`
-:   Replace the output file if it already exists, when building an
-    image. By default when building an image and an output artifact
-    already exists `mkosi` will refuse operation. Specify this option
-    once to delete all build artifacts from a previous run before
-    re-building the image. If incremental builds are enabled,
-    specifying this option twice will ensure the intermediary
-    cache files are removed, too, before the re-build is initiated. If a
-    package cache is used (also see the **Files** section below),
-    specifying this option thrice will ensure the package cache is
-    removed too, before the re-build is initiated. For the `clean`
-    operation this option has a slightly different effect: by default
-    the verb will only remove build artifacts from a previous run, when
-    specified once the incremental cache files are deleted too, and when
-    specified twice the package cache is also removed.
+:   Replace the output file if it already exists, by default `mkosi`refuses to overwrite an existing image.
+    This option can be specified once, twice or trice to change the scope of artifact removals.
+    Specify once to delete all build artifacts from a previous run. Twice to include removal of the
+    intermediary package cache from incremental builds. Thrice to remove all build artifacts and the the full
+    package cache. See the **Files** section for more context and details.
+    If `mkosi` is followed by the `clean` verb, the `-f` option can only be specified twice, see the
+    description of the command line verbs above.
 
 `--directory=`, `-C`
-:   Takes a path to a directory. `mkosi` switches to this directory before
-    doing anything. Note that the various configuration files are searched
-    for in this directory, hence using this option is an effective way to
-    build a project located in a specific directory.
+:   Specify a path to a directory. `mkosi` switches to this directory before doing anything. Note that the
+    various configuration files are searched for in this directory by default. Using this option can be used
+    to take advantage of defaults when executing mkosi inside of a profile directory or to execute `mkosi` in
+    any other directory deemed useful.
 
 `--debug`
-:   Enable additional debugging output.
+:   Enable additional debugging output. Also useful to observe from time to time when everything just works.
+    This pro-active approach leads to a better understanding on how `mkosi` works for future usage.
 
 `--debug-shell`
-:   When executing a command in the image fails, mkosi will start an interactive
-    shell in the image allowing further debugging.
+:   When executing a command that fails to build an image, mkosi will start an interactive shell inside of the
+    interrupted or failed image build. This feature helps to acquire additional insights for troubleshooting.
```

Comment:
Style: interrupted or failed, not both.
```suggestion
    interrupted or failed image build. This feature helps to acquire additional insights for troubleshooting.
```
The last sentence is too abstract, more concrete would be something like "This can be used to debug failing build scripts, see the **Scripts** section."

Reply:
ok I see, thank for the guidance, can you commit the change? I don't see your suggestion in the proposed correction.